tests/unit/test_logging.py

```diff
@@ -32,7 +32,10 @@ def test_filter_clears_exc_info_and_text():
 
 def test_setup_logging():
     """Test setting up logging."""
-    with patch("cou.logging.logging") as mock_logging:
+    with (
+        patch("cou.logging.logging") as mock_logging,
+        patch("cou.logging.pathlib"),
```

Comment:
I think that `COU_DATA` should be defined as auto fixture for whole unit tests, so we can be sure that no data are written to `/home/<user>/.local/share/cou`.

Reply:
Done